.github/workflows/build_all.yml

```diff
@@ -576,37 +588,38 @@ jobs:
         id: publish_artifacts
         if: github.event_name == 'push'
         env:
-          DOCKER_REGISTRY: "cwf-test.artifactory.magmacore.org"
-          DOCKER_USERNAME: "${{ secrets.JFROG_USERNAME }}"
-          DOCKER_PASSWORD: "${{ secrets.JFROG_PASSWORD }}"
+          DOCKER_REGISTRY: "linuxfoundation.jfrog.io/magma-docker-cwag-test"
+          DOCKER_USER: "${{ secrets.LF_JFROG_USERNAME }}"
+          DOCKER_PASSWORD: "${{ secrets.LF_JFROG_PASSWORD }}"
         run: |
           ./ci-scripts/tag-push-docker.sh --images 'cwag_go|gateway_go|gateway_python|gateway_sessiond|gateway_pipelined' --tag "${TAG}" --tag-latest true --project cwf
           ARTIFACTS="{\"packages\":[\"$DOCKER_REGISTRY/cwag_go:${TAG}\", \"$DOCKER_REGISTRY/gateway_go:${TAG}\", \"$DOCKER_REGISTRY/gateway_python:${TAG}\", \"$DOCKER_REGISTRY/gateway_sessiond:${TAG}\",\"$DOCKER_REGISTRY/gateway_pipelined:${TAG}\"],\"valid\":true}"
-          echo "::set-output name=artifacts::$(echo $ARTIFACTS)"
-      - name: Tag and push to Docker Registry goradius
-        if: github.event_name == 'push'
-        env:
-          DOCKER_REGISTRY: "${{ secrets.DOCKER_REGISTRY }}"
-          DOCKER_USERNAME: "${{ secrets.DOCKER_USERNAME }}"
-          DOCKER_PASSWORD: "${{ secrets.DOCKER_PASSWORD }}"
-        # yamllint disable rule:line-length
-        run: |
-          ./ci-scripts/tag-push-docker.sh --images 'goradius' --tag "${TAG}" --tag-latest true
-      - name: Tag and push to Docker Registry xwfm-integ-tests
-        if: github.event_name == 'push'
-        env:
-          DOCKER_REGISTRY: "${{ secrets.DOCKER_REGISTRY }}"
-          DOCKER_USERNAME: "${{ secrets.DOCKER_USERNAME }}"
-          DOCKER_PASSWORD: "${{ secrets.DOCKER_PASSWORD }}"
-        # yamllint disable rule:line-length
-        run: |
-          ./ci-scripts/tag-push-docker.sh --images 'xwfm-integ-tests' --tag "${TAG}" --tag-latest true
+          echo "artifacts=$(echo $ARTIFACTS)" >> $GITHUB_OUTPUT
+      # 2022-12-06 this is not supported in the new artifactory
```

Comment:
Do we really want to keep this as a comment or can we remove it?

Reply:
I think this is a relevant information for the 1.8 branch.